server/channels/utils/merge_test.go

```diff
@@ -1104,6 +1104,18 @@ func TestMergeWithVeryComplexStruct(t *testing.T) {
 		assert.Equal(t, patch, *merged)
 	})
 
+	t.Run("blank string in patch overwrites base string b/c empty string is not nil", func(t *testing.T) {
```

Comment:
First thought about using merge but discovered this little gotcha, so wanted to add a test case as documentation.